api/v1/swagger.yaml

```diff
@@ -1988,27 +1997,632 @@ paths:
                 type: array
                 items:
                   type: string
-              keywords:
-                description: Describes this generator.
+              tags:
+                type: array
+                items:
+                  type: string
+                  maxLength: 60
+                  pattern: ^[0-9A-Za-z_][0-9A-Za-z_\\-]{1,59}$
+                description: >
+                  Array of tag names. Tag name: Max 60 characters; case
+                  insensitive; allows alpha-numeric characters, underscore (_)
+                  and dash (-). Tag names cannot start with a dash (-).
+              connection:
+                type: object
+                description: Information about the connection.
+              context:
+                type: object
+                description: >-
+                  Variables accessible inside the transform function.
```

Comment:
they're also available inside the toUrl fn *and* for variable substitution in connection.url